### ISSUE TITLE
The 2nd argument of "trait_exists" function isn't recognized.

### ISSUE THREAD

**Issue by aik099** [scrutinizer-ci/php-analyzer#793]
The code:

```php
$isTrait = function_exists('trait_exists') ? trait_exists($class, $this->autoload) : false;
```

is reported as error:
```
The call to trait_exists() has too many arguments starting with $this->autoload.
```

Probably the PHP Analyzer has wrong `trait_exists` function declaration somewhere.

see https://scrutinizer-ci.com/g/php-annotations/php-annotations/issues/master/files/src/annotations/AnnotationManager.php?orderField=path&order=asc&issueId=0289906b-877b-4aee-a1aa-539991eb37f5
